blackjax/nuts.py

```diff
@@ -54,6 +59,7 @@ class NUTSInfo(NamedTuple):
     trajectory_rightmost_state: integrators.IntegratorState
     num_trajectory_expansions: int
     integration_steps: int
+    avg_accept_prob: float
```

Comment:
Would you mind calling this `acceptance_probability` for now (and leave a comment next to it)? I need this parameter in `Info` for step size adaptation. This is not perfect but it will do for now.

Reply:
Sure thing.